apps/dashboard_web/lib/dashboard_web/live/dashboard_live/layout.html.leex

```diff
@@ -1,52 +1,107 @@
+            <%= if length(component.configurations) > 0 do %>
+              <ul class="p-4">
+                <%= for configuration <- component.configurations do %>
+                  <li class="mb-2">
+                    <%= label f, :value, configuration.label, class: "text-gray-700 text-sm" %>
+                    <%= text_input f, "configuration_#{configuration.id}", class: "bg-gray-200 appearance-none border-2 border-gray-200 rounded py-1 px-2 text-gray-700 text-sm focus:outline-none focus:bg-white focus:border-blue-500 w-full" %>
+                    <%= hidden_input f, :has_config, value: true %>
+                  </li>
+                <% end %>
+              </ul>
+            <% else %>
+              <p class="italic p-4 text-gray-500 text-xs">
+                No configuration.
+              </p>
+            <% end %>
+          </div>
+          <div class="card-footer">
+            <%= submit "Add to dashboard", class: "btn btn-primary btn-sm flex-1" %>
+          </div>
+        </form>
+      <% end %>
+    </div>
   </div>
-</div>
 
-<div>
-  <h2>Active Components</h2>
-  <div id="sortable-container" phx-hook="ComponentDragAndDrop" style="display: flex; flex-direction: row; flex-wrap: wrap; border: 1px solid #eee;">
-    <%= for dc <- @dashboard_components do %>
-      <div style="background: white; display: flex; flex-direction: column; border: 1px solid #666; padding: 0.5em; margin: 0 0.5em 0.5em 0;" class="sortable" data-dc-id="<%= dc.id %>">
-        <div style="display: flex; flex-direction: row; align-items: center;">
-          <span phx-click="remove-component" phx-value-dc-id="<%= dc.id %>" style="cursor: pointer; color: red; margin-right: 0.2em; font-size: 1.2em;">-</span>
-          <%= dc.sequence %>: <%= dc.component.name %>
-        </div>
-        <%= for config <- dc.configurations do %>
-          <div>
-            <%= config.configuration.label %>: <%= config.value %>
+  <div class="">
+    <h2 class="font-bold mb-4 text-gray-800">Active Components</h2>
+    <div
+      class="bg-gray-200 grid grid-cols-5 gap-8 p-8 rounded-lg"
+      id="sortable-container"
+      phx-hook="ComponentDragAndDrop"
+    >
+      <%= for dc <- @dashboard_components do %>
+        <div
+          class="sortable card col-span-1 row-span-1"
+          data-dc-id="<%= dc.id %>"
+        >
+          <div class="card-header">
+            <h1 class="card-title">
+              <%= dc.component.name %>
+            </h1>
+          </div>
+          <div class="flex-1">
+            <%= if length(dc.configurations) > 0 do %>
+              <ul class="p-4">
+                <%= for config <- dc.configurations do %>
+                  <li class="text-gray-700 text-sm">
+                    <%= config.configuration.label %>: <%= config.value %>
+                  </li>
+                  <li class="text-gray-700 text-sm">
+                    <%= config.configuration.label %>: <%= config.value %>
+                  </li>
```

Comment:
Is there a reason this is in there twice?

Reply:
Whoops, testing purposes. 😅